package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@codahq/packs-sdk",
-  "version": "1.7.7-prerelease.2",
+  "version": "1.7.7-prerelease.3",
```

Comment:
I think we should just release. Otherwise you can not release new pack versions which I am sure we will want to do

Reply:
Yeah, just saw Jonathan's comment come through.  Can go to 1.7.8.

Reply:
@sam-codaio - updated to new version.